examples/gateways/example-gateway/main.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/brigadecore/brigade/sdk/v2"
+	"github.com/brigadecore/brigade/sdk/v2/core"
+	"github.com/brigadecore/brigade/sdk/v2/restmachinery"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Get the Brigade API server address and gateway token from the environment
+	apiServerAddress := os.Getenv("APISERVER_ADDRESS")
+	if apiServerAddress == "" {
+		log.Fatalf("Required environment variable APISERVER_ADDRESS not found.\n")
+	}
+	gatewayToken := os.Getenv("GATEWAY_TOKEN")
```

Comment:
imho, `GATEWAY_TOKEN` has the potential to be misleading because it could be interpreted as a token for authenticating _to_ the gateway rather than a token used _by_ the gateway to authenticate to the Brigade API server.

Any thoughts on changing to `API_TOKEN` or something of the sort?

Reply:
Apart from this, LGTM.

Dealer's choice on whether to act on this or merge.

Reply:
Agreed; updated.  Thanks!